File: Linguagem C++/dardo.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {

    double d1,d2,d3, maior;
    cout << "Digite as tres distancias: \n";
    cin >> d1 >> d2 >> d3;
    if (d1 > d2 && d1 > d3) {
        maior = d1;
    }
    else if (d2 > d3) {
        maior = d2;
    }
    else {
        maior = d3;
    }

    cout << fixed << setprecision(2);
    cout <<  "Maior distancia = " << maior;
    return 0;
}

File: Linguagem Python/EXs - Estrutura Condicional PY/dardo.py
print("Digite as tres distancias: ")
a = float(input())
b = float(input())
c = float(input())

if a > b and a > c:
    maior = a
elif b > c:
    maior = b
else:
    maior = c

print(f"Maior distancia = {maior:.2f}")